### PR TITLE
Create SubscribeForm.php

### PR DESCRIPTION
**Pull request by sandykurniawan19**
Create Subscribe Validation , for adding sirius

### CODE REVIEW

File: components/SubscribeForm.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php  namespace App\Components\Forms;
```
Comment (jayvdb) [inline components/SubscribeForm.php:1]: namespace should be on a new line.

**Comment by zhiephie** [comment]
Do not create files in the folder components, folder components alias views

File: components/SubscribeForm.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+use App\Components\Validation\FormValidator;
```
Comment (jayvdb) [inline components/SubscribeForm.php:3]: This depends on https://github.com/remluben/slim-boilerplate/tree/master/app/Components/Validation ? which is not included.

**Comment by jayvdb** [comment]
> Do not create files in the folder components, folder components alias views

Do you want this form?
It [is MIT](https://github.com/remluben/slim-boilerplate/blob/master/LICENSE)
like
[this repo](https://github.com/zhiephie/boilerplate-slim3/blob/9abad89a30b58700d11eb9552dd886255ad9b0c9/composer.json#L6).

It should be moved to
https://github.com/zhiephie/boilerplate-slim3/tree/master/app/src/Components/Forms
or
https://github.com/zhiephie/boilerplate-slim3/tree/master/app/src/Forms
?

**Comment by zhiephie** [comment]
to 
https://github.com/zhiephie/boilerplate-slim3/tree/master/app/src/Forms

**Comment by sandykurniawan19** [comment]
thanks for correct mr @jayvdb

**Pull request merged**